app/config/AppConfig.scala

```diff
@@ -55,9 +55,23 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig) {
 
   private def requestUri(implicit request: RequestHeader): String = SafeRedirectUrl(appUrl + request.uri).encodedUrl
 
-  def feedbackUrl(implicit request: RequestHeader): String = {
+  def betaFeedbackUrl(implicit request: RequestHeader): String = {
     s"$contactFrontEndUrl/contact/beta-feedback?service=$contactFormServiceIdentifier&backUrl=$requestUri"
   }
 
+  private lazy val feedbackFrontendUrl = {
+    val feedbackSurveyUrl = servicesConfig.baseUrl("feedback-frontend")
+    servicesConfig.getConfString("feedback-frontend.relativeUrl", feedbackSurveyUrl)
+  }
+
+  lazy val feedbackSurveyUrl: String = s"$feedbackFrontendUrl/feedback/$contactFormServiceIdentifier"
+
   lazy val contactUrl = s"$contactFrontEndUrl/contact/contact-hmrc?service=$contactFormServiceIdentifier"
+
+  private lazy val basGatewayUrl = {
+    val basGatewayUrl = servicesConfig.baseUrl("bas-gateway")
+    servicesConfig.getConfString("bas-gateway.relativeUrl", basGatewayUrl)
```

Comment:
think these should be bas-gateway-frontend

Reply:
Good spot. Does this mean the sign out url should be s"$basGatewayUrl/**bas-gateway-frontend**/sign-out-without-state instead of s"$basGatewayUrl/bas-gateway/sign-out-without-state ?

Reply:
rest of the URL is okay, see https://github.com/hmrc/bas-gateway-frontend/blob/master/conf/prod.routes#L2